src/components/events/WindowResize.ts

```diff
@@ -0,0 +1,15 @@
+/**
+ * Fired when window is resized
+ */
+export const WindowResize = 'window resize';
```

Comment:
event name should be in a past form
```suggestion
export const WindowResized = 'window resized';
```